torch/csrc/jit/interpreter.cpp

```diff
@@ -353,14 +457,25 @@ struct CodeImpl {
     return value_to_reg_.at(v);
   }
 
-  void emitLoadInputs(Node* node) {
-    auto move_flag_it = preprocess_.move_flags.at(node).begin();
-    for (Value* input : node->inputs()) {
+  void emitUse(Value* input, bool drop) {
+    // drop - if true, we are not actually going to use this thing
+    // and we can short circuit doing many instructions here
+    // by either clearing the register (DROPR) or just popping the stack
+    // (DROP)
+    if (preprocess_.can_emit_inline[input->node()]) {
```

Comment:
can you add a comment about what the significant of the check to can_emit_inline is here